backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -643,11 +643,34 @@ export class ExperimentService {
 
         experimentDoc.experimentSegmentInclusion = oldExperiment.experimentSegmentInclusion;
         experimentDoc.experimentSegmentExclusion = oldExperiment.experimentSegmentExclusion;
+
+        const currentIncludeData = experimentSegmentInclusion.segment
```

Comment:
can we refactor into a reusable function so we are not duplicating code?

Reply:
Created a new function for both include & exclude